app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -370,8 +370,6 @@ ShortFormApplicationController = (
   $scope.checkAfterLiveInTheNeighborhood = (preference) ->
     # preference is either neighborhoodResidence or antiDisplacement
     if ShortFormApplicationService.applicationHasPreference(preference)
-      # Neighborhood provides automatic liveInSf preference
-      ShortFormApplicationService.copyNeighborhoodToLiveInSf(preference)
```

Comment:
@chrisdolendo it is _almost_ redundant but not entirely, just thinking of edge cases... like if the person filled out Jane Doe + uploaded proof, all that would get copied over. If they then switched the dropdown to Jim Smith, that wouldn't get copied over to Live in SF. So we'd either need to add a callback to that dropdown or keep this one..... actually yes maybe a callback on the member dropdown is actually preferred because it would solve the case where they change the dropdown but jump around in the nav without hitting "Next".